src/plotrecipes/distplot1D.jl

```diff
@@ -21,10 +21,8 @@
 
   # plot histogram
   @series begin
-    seriestype --> :step
-    seriescolor --> :black
     h
-  end
+   end
```

Comment:
perhaps there is a misalignment here, can you please double check?

Reply:
Ahh yes, it's fixed now